### PR TITLE
R/app

### PR DESCRIPTION
**Pull request by danriedl**
when marge?

### CODE REVIEW

File: docker-git-sync.py [hunk 1]
```diff
@@ -1,133 +1,169 @@
...
             logging.info(result)
 
 
-def find_yaml_config(args):
+def find_yaml_config(args) -> None:
     # For loop to search for yaml file in servers.
-    for file in walk_through_files(path=args.server_directory):
-        content = yaml.safe_load(open(file))
+    server_directory = Path(args.server_directory)
+    for file in server_directory.rglob("*.yaml") + server_directory.rglob("*.yml"):
```
Comment (pat-git) [inline docker-git-sync.py]: This does not work for me: 
```
Traceback (most recent call last):
  File "/mnt/f/workspace/docker-compose-ops/docker-git-sync.py", line 171, in <module>
    main()
  File "/mnt/f/workspace/docker-compose-ops/docker-git-sync.py", line 167, in main
    find_yaml_config(args)
  File "/mnt/f/workspace/docker-compose-ops/docker-git-sync.py", line 102, in find_yaml_config
    for file in server_directory.rglob("*.yaml") + server_directory.rglob("*.yml"):
TypeError: unsupported operand type(s) for +: 'generator' and 'generator'
```

Possible fix:
```suggestion
    from itertools import chain
    
    for file in chain(server_directory.rglob("*.yaml"), server_directory.rglob("*.yml")):
```

**Review by pat-git — CHANGES_REQUESTED** [review]
Nice work 👍
see small changes about the issue with glob generator join (+ operator does not work)

File: docker-git-sync.py (hunk 1 above)
Comment (danriedl) [reply docker-git-sync.py]: TIL. Good catch 🐈‍⬛

**Review by pat-git — APPROVED** [review]
👍

**Pull request merged**